src/handlers/notify_zulip.rs

```diff
@@ -12,6 +12,8 @@ pub(super) struct NotifyZulipInput {
     /// For example, if an `I-prioritize` issue is closed,
     /// this field will be `I-prioritize`.
     label: Label,
+    is_default_valid: bool,
+    names: Vec<String>,
```

Comment:
It's not clear at first glance what both of them do, a comment for each of them would be appreciated.

Reply:
Done.